bitbots_vision_tools/scripts/colorpicker.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
+
+import cv2
+import numpy as np
+import time
+import pickle
+import os
+import rospy
+import VisionExtensions
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+
+rospy.init_node('bitbots_colorpicker')
+rospy.loginfo('Initializing colorpicker...', logger_name="colorpicker")
+_cv_bridge = CvBridge()
+
+
+def click_and_crop(event, x, y, flags, param):
+	"""
+	Callback for the OpenCV cursor.
+
+	:param event: Event type
+	:param x: Mouse x position
+	:param y: Mouse y position
+	:param flags: Callback flags
+	:param param: Some unused parameter
+	"""
+	global coord, left_click, undo_click
+
+	if flags == (cv2.EVENT_FLAG_SHIFTKEY + cv2.EVENT_FLAG_LBUTTON):
+		# Substracts colors
+		undo_click = True
+	elif event == cv2.EVENT_LBUTTONUP:
+		# Adds colors
+		left_click = True
+
+	# Set coordinates
+	coord = (x, y)
+
+
+def draw_mask(image, mask, color, opacity=0.5):
+	"""
+	Draws the mask on an image
+
+	:param image: The image canvas
+	:param mask: The binary mask
+	:param color: Color bgr tuple
+	:param opacity: Opacity of the mask
+	"""
+	# Make a colored image
+	colored_image = np.zeros_like(image)
+	colored_image[:, :] = tuple(np.multiply(color, opacity).astype(np.uint8))
+
+	# Compose debug image with lines
+	return cv2.add(cv2.bitwise_and(image, image, mask=255-mask),
+			cv2.add(colored_image*opacity, image*(1-opacity), mask=mask).astype(np.uint8))
+
+
+def init_text():
+	"""
+	Prints some usage information.
+	"""
+	rospy.loginfo( "Welcome to the Colorpicker\n\
+Click to select the colors in the OpenCV window.\n\
+Use <shift> and <left click> to remove colors.\n\
+To undo a step press 'u'.\n\
```

Comment:
```suggestion
To undo a step press 'u'.\n\
To increase the size of the selection box press '+', to decrease press '-' \n\
```

Reply:
It is indeed intended to use the new line characters here. This improves the readability in the terminal.

Reply:
My suggested change was not supposed to change anything about the new lines. It was only supposed to add the second line which explains the '+' and '-' buttons.

Reply:
Ah okay.